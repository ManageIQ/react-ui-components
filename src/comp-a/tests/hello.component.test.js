import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import { HelloCmp } from '../some-cmp/hello.component';

test('Some dummy test', () => {
  const component = renderer.create(<HelloCmp />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Method test example', () => {
  const component = shallow(<HelloCmp />);
  expect(component.instance().bla()).toBe('some text');
});
