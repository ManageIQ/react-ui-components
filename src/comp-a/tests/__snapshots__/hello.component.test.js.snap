// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some dummy test 1`] = `
<div
  className="some-class"
>
  <div
    className="alert alert-danger"
  >
    <span
      aria-hidden="true"
      className="pficon pficon-error-circle-o"
    />
    I am an Alert
  </div>
</div>
`;
